crates/ruff_python_ast/src/visitor/preorder.rs

```diff
@@ -288,12 +288,17 @@ where
         Stmt::If(ast::StmtIf {
             test,
             body,
-            orelse,
+            elif_else_clauses,
             range: _range,
         }) => {
             visitor.visit_expr(test);
             visitor.visit_body(body);
-            visitor.visit_body(orelse);
+            for clause in elif_else_clauses {
+                if let Some(test) = &clause.test {
+                    visitor.visit_expr(test);
+                }
+                visitor.visit_body(&clause.body);
```

Comment:
Same as for `Visitor`:

* Add an `visit_elif_else_clause` method
* Call `walk_elif_else_clause` here

Reply:
```suggestion
                visitor.visit_elif_else_clause(visitor, clause);
```

Reply:
apparently i already added the `visit_elif_else_clause`, i just forgot to use it. i get confused by all the different visitors.